Assignment/solution_student/index.html

```diff
@@ -8,24 +8,88 @@
     <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
     <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
 
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,600,800' rel='stylesheet' type='text/css'>
+
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
   <body>
 
     <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
-    
-      <header>
-        <h1>FEWD HTML Template</h1>
+
+      <header class="clearfix">
+        <div class="header-content">
+          <div class="logo">Relaxr</div>
+          <nav>
+            <ul>
+              <li><a href="#">About</a></li>
+              <li><a href="#">FAQ</a></li>
+              <li><a href="#">Team</a></li>
+              <li><a href="#">Contact Us</a></li>
+              <li><a href="#">Blog</a></li>
+            </ul>
+          </nav>
+        </div> 
       </header>
 
+      <div class="main-content">
       <main>
-        <p>Our HTML starter template!</p>
+        <section class="article-container">
```

Comment:
Nothing major, but I'd caution against using the `<section>` element in this way (also on lines 78 and 79), because it seems to me like it's being used more as a container for styling then to semantically denote a section of content — if the line between the two seems a little blurry, that's because it kind of is, but check out this article that might shed some light on how to determine when to use a div and when to use a section: http://html5doctor.com/avoiding-common-html5-mistakes/section-wrapper